cmd/kubelet/app/init_windows.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"k8s.io/kubernetes/cmd/kubelet/app/options"
+	"k8s.io/kubernetes/pkg/windows/service"
+)
+
+const (
+	serviceName = "kubelet"
+)
+
+func initForOS() error {
```

Comment:
When you remove the global `FlRunService` var in favor of `KubeletFlags.WindowsService`, you'll have to pass it in as an arg here, e.g. `func initForOS(windowsService bool) error`.

I know that's not as pretty a function signature, but it's better than global state floating around and keeps things consistent with the other flags.

Reply:
Done